vividus-extension-selenium/src/main/java/org/vividus/ui/util/LocatorConversionUtils.java

```diff
@@ -92,8 +95,19 @@ private void applyFilter(Locator locator, String filterType, String filterValue)
 
     private Locator convertToLocator(String searchType, String searchValue)
     {
-        return createLocator(elementActionService.getSearchLocatorTypes(),
-            type -> new Locator(type, searchValue), "locator", searchType);
+        Function<LocatorType, Locator> mapper;
+        if ("shadowcssselector".equals(searchType))
```

Comment:
the problem here is that general-purpose `LocatorConversionUtils` (used for both web ui and mobile native apps) "knows" about shadow DOM which is web ui specific